hack/cluster-deploy.sh

```diff
@@ -43,17 +43,8 @@ function _deploy_infra_for_tests() {
 }
 
 function _ensure_cdi_deployment() {
-    _kubectl apply -f - <<EOF
----
-apiVersion: cdi.kubevirt.io/v1beta1
-kind: CDI
-metadata:
-  name: ${cdi_namespace}
-spec:
-  config:
-    featureGates:
-    - HonorWaitForFirstConsumer
-EOF
+    # enable featuregate
+    _kubectl patch cdi ${cdi_namespace} --type merge -p '{"spec": {"config": {"featureGates": [ "HonorWaitForFirstConsumer" ]}}}'
```

Comment:
I did not expect that my PR goes in before yours. I think what we need to modify is that we also add two insecure registries in this patch call (https://github.com/kubevirt/kubevirt/pull/6234/files#diff-be559fd8e5c86cd4bfb53d341659b318f4c5963a8da53ca413c19528590edd4bR58), or in an additional patch, instead of adding the single insecure registry further down below.

Reply:
Thanks for the heads up!

Reply:
I updated the PR, can you please take a look? Thanks!